plugins/examples/rescore/src/main/java/org/elasticsearch/example/rescore/ExampleRescoreBuilder.java

```diff
@@ -173,10 +173,10 @@ public TopDocs rescore(TopDocs topDocs, IndexSearcher searcher, RescoreContext r
                             endDoc = leaf.docBase + leaf.reader().maxDoc();
                         } while (topDocs.scoreDocs[i].doc >= endDoc);
                         LeafFieldData fd = context.factorField.load(leaf);
-                        if (false == (fd instanceof LeafNumericFieldData leafNumericFieldData)) {
+                        if (false == (fd instanceof LeafNumericFieldData)) {
```

Comment:
This looks like an unintentional revert? Or is it intentional?

Reply:
Intentional, this change broke our example plugins so I just decided to fix it in this PR.